common/numeric-constants.ts

```diff
@@ -4,3 +4,5 @@ export const NUMERIC_FIXED_VAR = 0.005
 export const NUMERIC_GRAPH_COLOR = '#5fa5f9'
 export const NUMERIC_TEXT_COLOR = 'text-blue-500'
 export const UNIQUE_BETTOR_BONUS_AMOUNT = 10
+export const BETTING_STREAK_BONUS_AMOUNT = 1
```

Comment:
![](https://www.pewpewtactical.com/wp-content/uploads/2020/05/You-gotta-pump-those-numbers-up-those-are-rookie-numbers.gif)

Reply:
I'd raise this to M$5 or M$10 probably...

Reply:
🤣